experimental/dds/tree/src/test/EditLog.tests.ts

```diff
@@ -338,6 +338,53 @@ describe('EditLog', () => {
 		expect(editsEvicted).to.equal(targetEditLogSize * 2 - collaborationWindowSize);
 	});
 
+	it("can handle sparse sequence numbers with a minimum sequence number that's not in memory", () => {
+		const targetEditLogSize = 10;
+		const log = new EditLog(undefined, undefined, undefined, targetEditLogSize, targetEditLogSize * 2);
+		const ids: EditId[] = [];
```

Comment:
This is the test for when our current _minSequenceNumber is not in the map.

Reply:
Sorry, I just mean the variable here named `ids`. You push to it below but you never read it or anything.

Reply:
Ohh I see. I thought this got past CI but now that I'm looking, it seems like a bunch of build steps were skipped for some reason...

Reply:
Oh never mind, client packages is the only one changed.

Reply:
Yea the variable is technically being used, because you pushed to it, so lint isn't going to catch it. After all, who's to say that you didn't override `Array.prototype.push` to have some interesting side effect? That's normal and good practice, right? :D